### PR TITLE
fix(args): args should be passed with exec or cli_args

### PR DESCRIPTION
**Pull request by paologallinaharbur**
Fix https://github.com/newrelic/newrelic-prometheus-exporters-packages/issues/74

### CODE REVIEW

File: nri-config-generator/main.go [hunk 1]
```diff
@@ -139,3 +144,10 @@ func loadConfigTemplate() (*template.Template, error) {
 	}
 	return t, nil
 }
+
+func prometheusExportersBinPath(name string) string {
+	if runtime.GOOS == "windows" {
+		return filepath.Join("C:\\Program Files\\Prometheus-exporters\\bin", fmt.Sprintf("%s.exe", name))
```
Comment (gsanchezgavier) [inline nri-config-generator/main.go:150]: woud make sense also to move the win path to a constant next to the `nixExporterBinPath` ?

File: exporters/powerdns/powerdns.json.tmpl [hunk 2]
```diff
@@ -1,8 +1,21 @@
 {
-    "arguments": {
```
Comment (gsanchezgavier) [inline exporters/powerdns/powerdns.json.tmpl:2]: is wired that this was working in the first place, i guess the agent also is using that but in an unsupported way or something like that

**Review by gsanchezgavier — APPROVED** [review]
thanks for spotting this and fix it! i just left an small suggestion.

File: nri-config-generator/main.go (hunk 1 above)
Comment (paologallinaharbur) [reply nri-config-generator/main.go:150]: totally

**Pull request merged**